packages/bridge-ui-v2/src/components/ChainSelector/ChainSelector.svelte

```diff
@@ -1,15 +1,20 @@
 <script lang="ts">
-  import type { Chain } from '@wagmi/core';
+  import type { Chain, GetNetworkResult } from '@wagmi/core';
   import type { ComponentType } from 'svelte';
   import { noop, onDestroy } from 'svelte/internal';
   import { t } from 'svelte-i18n';
 
   import { EthIcon, Icon, TaikoIcon } from '$components/Icon';
+  import { warningToast } from '$components/NotificationToast';
   import { PUBLIC_L1_CHAIN_ID, PUBLIC_L2_CHAIN_ID } from '$env/static/public';
   import { chains } from '$libs/chain';
   import { uid } from '$libs/util/uid';
+  import { account } from '$stores/account';
+
+  // TODO: think about updating the state for the network here
 
   export let label: string;
+  export let value: Maybe<GetNetworkResult['chain']> = null;
```

Comment:
nit: can we do a better name than `value`? perhaps `selectedChain`

Reply:
We could, but this is the value of this control, same as Input, same as Select, same as Textarea, etc... I find it a bit redundant to use `selectedChain` in a ChainSelected component:
`<ChainSelector selectedChain={chain} />` vs `<ChainSelector value={chain} />`